admob/integration_test/src/integration_test.cc

```diff
@@ -716,22 +742,29 @@ TEST_F(FirebaseAdMobTest, TestInterstitialAdLoadAndShow) {
 
   LogDebug("Click the Ad, and then return to the app to continue");
 
-  while (
-      full_screen_content_listener.num_on_ad_dismissed_full_screen_content_ ==
-      0) {
+  while (content_listener.num_ad_dismissed() == 0) {
     app_framework::ProcessEvents(1000);
   }
 
   LogDebug("Waiting for a moment to ensure all callbacks are recorded.");
   app_framework::ProcessEvents(2000);
 
-  EXPECT_EQ(full_screen_content_listener.num_on_ad_clicked_, 1);
-  EXPECT_EQ(full_screen_content_listener.num_on_ad_showed_full_screen_content_,
-            1);
-  EXPECT_EQ(full_screen_content_listener.num_on_ad_impression_, 1);
-  EXPECT_EQ(paid_event_listener.num_on_paid_event_, 1);
-  EXPECT_EQ(
-      full_screen_content_listener.num_on_ad_dismissed_full_screen_content_, 1);
+  EXPECT_EQ(content_listener.num_ad_clicked(), 1);
+  EXPECT_EQ(content_listener.num_ad_showed_content(), 1);
+  EXPECT_EQ(content_listener.num_ad_impressions(), 1);
+  EXPECT_EQ(content_listener.num_ad_failed_to_show_content(), 0);
+  EXPECT_EQ(content_listener.num_ad_dismissed(), 1);
+  EXPECT_EQ(paid_event_listener.num_paid_events(), 1);
+
+#if (defined(TARGET_OS_IPHONE) && TARGET_OS_IPHONE)
+  // Show the Ad again.  Note: Android's Interstitial ads fail silently
+  // when attempting to show the ad twice.
```

Comment:
Reaching out to the Android GMA team as to why this is true.

Reply:
Looks like this is a new feature.  I'll update the dependency tomorrow morning and try it again.